packages/client/manifest.json

```diff
@@ -1482,6 +1483,56 @@
           "Palette 10"
         ]
       },
+      {
+        "type": "color",
+        "label": "C1",
+        "key": "c1",
+        "barSeparator": false,
```

Comment:
NAB: not an issue but more an FYI - `barSeparator` only needs specified if you define `showInBar: true` which means use the settings bar that floats in the app preview. If you enable the bar, then `barSeparator: true` means render a divider after a certain setting. So basically all these usages of `barSeparator` here do nothing and so can be deleted.

Reply:
Cheers, good to know! I've removed them.